tests/src/FunctionalJavascript/ModuleIntegrationTest.php

```diff
@@ -393,4 +395,182 @@ public function testDevicePreview() {
     $this->assertTrue($this->compareScreenToImage($this->getScreenshotFile('test_device_preview_ar12')));
   }
 
+  /**
+   * Testing integration of "liveblog" module.
+   */
+  public function testLiveblog() {
+    $pusherCredentials = json_decode(getenv('PUSHER_CREDENTIALS'), TRUE);
+    if (empty($pusherCredentials)) {
+      $this->fail("pusher credentials not provided.");
+      return;
+    }
+    if (!\Drupal::service('module_installer')->install(['thunder_liveblog'])) {
+      $this->fail("liveblog module couldn't be installed.");
+      return;
+    }
+
+    // Configure Pusher.
+    $this->logWithRole('administrator');
+
+    $page = $this->getSession()->getPage();
+    $this->drupalGet('admin/config/content/liveblog');
+
+    $fieldValues = [
+      'plugin_settings[app_id]' => $pusherCredentials['app_id'],
+      'plugin_settings[key]' => $pusherCredentials['key'],
+      'plugin_settings[secret]' => $pusherCredentials['secret'],
+      'plugin_settings[cluster]' => $pusherCredentials['cluster'],
+    ];
+    $this->setFieldValues($page, $fieldValues);
+    $this->click('input[data-drupal-selector="edit-submit"]');
+
+    $this->waitUntilVisible('.messages--status');
+
+    $this->logWithRole(static::$defaultUserRole);
+
+    // Add liveblog node.
+    $fieldValues = [
+      'title[0][value]' => 'Test Liveblog',
+      'field_highlights[values][3]' => 'element',
+      'field_posts_number_initial[0][value]' => '1',
+    ];
+
+    $this->drupalGet('node/add/liveblog');
+    $this->assertSession()->assertWaitOnAjaxRequest();
+    $this->setFieldValues($this->getSession()->getPage(), $fieldValues);
+    // 1 saves it as published in this case.
+    $this->clickArticleSave(1);
+
+    // Add first post.
+    $page = $this->getSession()->getPage();
+
+    $this->liveblogSetTitle($page, 'Normal post');
+    $this->liveblogSetBody($page, "This is a normal text");
+    $this->liveblogSubmit();
+
+    $this->waitUntilVisible('article[data-postid="1"]', 10000);
+
+    // Add post with image.
+    $this->liveblogSetTitle($page, 'Image post');
+
+    $this->clickDropButton('field_embed_media_image_add_more', FALSE);
+
+    $this->selectMedia("field_embed_media_0_subform_field_image", 'image_browser', ['media:1']);
+
+    $this->liveblogSetBody($page, 'Very nice image post you have here!');
+
+    $this->liveblogSubmit();
+
+    $this->waitUntilVisible('article[data-postid="2"]', 10000);
+    $this->waitUntilVisible('article[data-postid="2"] img.b-loaded', 10000);
+
+    // Add post with twitter.
+    $this->liveblogSetTitle($page, 'Twitter post');
+
+    $this->clickDropButton('field_embed_media_twitter_add_more');
+    $this->setFieldValue($page,
```

Comment:
Yes, since paragraphs are between title and body of Live Blog. That's true. Good point. 👍 
I was thinking mainly to get rid of one line wrapping functions, that's all.